docs/conf.py

```diff
@@ -212,14 +212,34 @@
         # A RTD thing to avoid a CSRF block when docs are using a custom domain
         hoverxref_api_host = "/_"
 
-hoverxref_tooltip_maxwidth = 600  # RTD main window is 696px
-hoverxref_auto_ref = True
-hoverxref_mathjax = True
-# hoverxref has to be applied to these
+hoverxref_auto_ref = False
 hoverxref_domains = ["py"]
-
+hoverxref_mathjax = True
+hoverxref_modal_hover_delay = 500
+hoverxref_tooltip_maxwidth = 600  # RTD main window is 696px
+hoverxref_intersphinx = [
+    "astropy",
+    "matplotlib",
+    "numpy",
+    "python",
+    "scipy",
+    "hvpy",
+    "asdf",
+    "zeep",
```

Comment:
Do we really need this?

Reply:
Why not? Surely we want this to work for links that are API and we link to zeep classes.